src/property.rs

```diff
@@ -17,7 +17,6 @@ pub trait Property: Send + Sync {
         let mut description = self.get_metadata();
         description.remove("@type");
         description.remove("unit");
-        description.remove("label");
```

Comment:
I believe this should gain `description.remove("title")` so that the validator isn't sad later on (which might be what's causing the travis failures)